### PR TITLE
Issue #15329: updated coverage table for section 4.1.2, added blue tick for LeftCurly module and a message explaining the reason

### PR DESCRIPTION
**Pull request by Zopsss**
#15329

Linked #15324 

Updated coverage table of google style guide implementation for section [4.1.2 Nonempty blocks: K & R style](https://checkstyle.org/styleguides/google-java-style-20180523/javaguide.html#s4.1.2-blocks-k-r-style), added blue tick for "LeftCurly" module and a message referencing the issue.

### CODE REVIEW

**Comment by Zopsss** [comment]
GitHub, generate site

**Comment by Zopsss** [comment]
Website: https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/3f82ba6_2024200511/google_style.html#a4.1.2

**Review by romani — CHANGES_REQUESTED** [review]
commit should reference https://github.com/checkstyle/checkstyle/issues/15329
issue.

issue in commit prefix should mean fix for this issue.

File: src/xdocs/google_style.xml
Comment (romani) [inline src/xdocs/google_style.xml:671]: in this comment https://github.com/checkstyle/checkstyle/pull/14901/files#r1660867034
we reference different issue.
please clarify.

Comment (Zopsss) [reply src/xdocs/google_style.xml:671]: Both issues are similar and refers to the same problem, my bad I shouldn't have created new issue for this.

Comment (Zopsss) [reply src/xdocs/google_style.xml]: changed the issue reference:

https://github.com/checkstyle/checkstyle/blob/57dfc92e25024546e39d27870f70a6edca707f59/src/xdocs/google_style.xml#L669-L671

File: src/xdocs/google_style.xml
Comment (romani) [inline src/xdocs/google_style.xml:669]: Will be addressed.

**Review by romani — APPROVED** [review]
Ok to merge, if we fix this before merge it will be good

File: src/xdocs/google_style.xml
Comment (Zopsss) [reply src/xdocs/google_style.xml]: fixed.

**Review by romani — APPROVED** [review]
ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Comment by rdiachenko** [comment]
@Zopsss , please fix ci `Check PR Description / Analyze (pull_request)`

**Comment by romani** [comment]
PR description is updated

**Pull request merged**
